vehicle_info:
  vehicle_mass: 2.7 # done. chassis and battery (probably total)
  maximum_steering_angle: 30.0
  minimum_turning_radius: 0.462
  wheel_base: 0.265
  wheel_radius: 0.0342
  wheel_width: 0.025 # not sure
  #wheel_tread: 1.55
  vehicle_length: 0.425 # done
  vehicle_width: 0.192 # done
  vehicle_height: 0.182 # done

  # Add parameters needed for my Intro to Robust Controls project below (lf, lr, cf, cr, M0, Iz0)
